Tests/Rewriters/MockInstantiationRewriterTests.cs

```diff
@@ -42,9 +41,9 @@ public void SetUp ()
     [TestCase ("_mock = MockRepository.GenerateMock<string, IDisposable, IConvertible>();", "_mock = new Mock<string>();")]
     [TestCase ("_mock = MockRepository.GeneratePartialMock<string> (42);", "_mock = new Mock<string> (42);")]
     [TestCase ("_mock = MockRepository.GeneratePartialMock<string>();", "_mock = new Mock<string>();")]
-    [TestCase ("_mock = MockRepository.GenerateStrictMock<string>();", "_mock = new Mock<string> (MockBehaviour.Strict);")]
-    [TestCase ("_mock = MockRepository.GenerateStrictMock<string> (42);", "_mock = new Mock<string> (MockBehaviour.Strict, 42);")]
-    [TestCase ("_mock = MockRepository.GenerateStrictMock<string, IDisposable>();", "_mock = new Mock<string> (MockBehaviour.Strict);")]
+    [TestCase ("_mock = MockRepository.GenerateStrictMock<string>();", "_mock = new Mock<string> (MockBehavior.Strict);")]
```

Comment:
Interesting. The typo fix should be a separate commit on develop as it does not fit into this feature.